clients-src/modules/ROOT/pages/console.adoc

```diff
@@ -499,4 +505,136 @@ Transaction closed without committing changes
 Database 'test' deleted
 ----
 
-//#todo Add compatibility info
+[#_version_compatibility]
+== Version Compatibility
+
+[cols="^.^2,^.^1,^.^2,^.^2"]
+|===
+| TypeDB Console | Protocol version | TypeDB | TypeDB Cloud & TypeDB Enterprise
```

Comment:
The values below look correct to me. However, what's the rationale for the order of the columns?

Reply:
It was originally like that in other compatibility tables (except the database encoding one).
I think it's like this: The first column should be the product we are talking about on the page. 
The second is for the Protocol version since it's directly connected with the product version but is nowhere else to be found explicitly.
The rest are for the server version we are compatible to. Core first (we didn't rename it yet in these, but it will happen soon. If you want, I can implement the renaming in these tables now).
If it's needed, at the very end we might add some other compatibility info, like framework version.

Reply:
If you're happing that there's a reason to the ordering, then I'm happy; but I didn't pick up on it myself.